keps/sig-cluster-lifecycle/0014-20180707-componentconfig-api-types-to-staging.md

```diff
@@ -0,0 +1,297 @@
+---
+kep-number: 14
+title: Moving ComponentConfig API types to staging repos
+status: provisional
+authors:
+  - "@luxas"
+  - "@sttts"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-api-machinery
+  - sig-node
+  - sig-network
+  - sig-scheduling
+  - sig-cloud-provider
+reviewers:
+  - "@thockin"
+  - "@liggitt"
+  - "@wojtek-t"
+  - "@stewart-yu"
+  - "@dixudx"
+approvers:
+  - "@thockin"
+  - "@jbeda"
```

Comment:
This should really be someone from api-machinery b/c it would be a updated policy.

Reply:
@lavalamp as well?

Reply:
@lavalamp is out of office for some time. Would add @deads2k instead.